src/sagemaker_tensorflow_container/training.py

```diff
@@ -192,6 +192,10 @@ def main():
     """
     hyperparameters = framework.env.read_hyperparameters()
     env = framework.training_env(hyperparameters=hyperparameters)
+
+    if '_tuning_objective_metric' in hyperparameters:
```

Comment:
I'd add a comment similar to https://github.com/aws/sagemaker-tensorflow-container/blob/master/src/tf_container/train_entry_point.py#L118-L119 about why we have this logic

Reply:
Added.